exampleCourse/questions/demo/annotated/engCircuit/server.py

```diff
@@ -1,40 +1,77 @@
 import random
+import io
 
+import matplotlib.pyplot as plt
+from schemdraw import Drawing
+import schemdraw.elements as elm
+
+def file(data):
+    if data["filename"] == "figure.png":
+        drawing = Drawing()
+        R1 = str(data["params"]["R1"]) + " $\\Omega$"
```

Comment:
```suggestion
        R1 = str(data["params"]["R1"]) + r" $\Omega$"
```
Nit: same with other lines below.